content/essentials/intro-to-reactivity.mdx

```diff
@@ -138,34 +138,45 @@ This keeps the user interface and other system components synchronized with the
+This interaction leads to the accessor subscribing to the signal, establishing a dynamic connection.
+Every interaction between the signal and its accessor reinforces this subscription, guaranteeing that the associated effects are re-executed when the signal changes.
+* Both memos *and* effects have the ability to subscribe to signals.
+However, interactions with a signal outside of these won't initiate tracking, due to the absence of a subscriber, thus limiting the scope of reactivity.
+* The reactive system is built on the interplay between elements that can change (signals) and those that can be updated (effects and memos).
+The principles of tracking are not limited to these elements, however, but also extend to reactive props and store values.
+* Entities like store values and reactive props are inherently depdendent on signals.
+A "signal" stands as the only entity that operates autonomously and is reliably dependable.
 
-- Whenever an effect is created and accesses a signal, it establishes a subscription to that signal. 
-This means that the effect will be re-executed whenever the signal changes.
 
-- Tracking is scoped to effects. 
-If you access a signal outside the context of an effect, no tracking or subscription will occur. 
-This is because there is no effect subscribed to that signal.
+Using Solid:
+```tsx
+function Counter() {
+  const [count, setCount] = createSignal(0);
+  const increment = () => setCount(count() + 1); 
 
-- The tracking concept extends to reactive props and store values, as well. 
-When you access a reactive value within an effect, it triggers tracking and subscription just like within signals.
+  createEffect(() => {
+    console.log("My effect says " + count());
+  })
+    /* With the `createEffect`, the `count()` (the signal) is now tracked as a subscription of this effect. This code will run *only* when `count()` changes. */
 
-Within tracking, there are some key behaviours and tradeoffs that have to be considered:
+  console.log(count()); // In this case,`count()` is *not* in a trackintg scope. This means it won't re-run when `count()` changes.
 
-1. All reactivity is tracked rom function calls directly, or hidden in the getter and triggered by the property's access. 
-This means that where you access properties on reactive objects is important.
 
-2. Components and callbacks from control flows are not tracking scopes and are only executed **once**. 
-This means that destructuring or doing logic at the top-level of your components will **not re-execute**. 
-These signals, stores, and props must be accessed from within their reactive primitives, or within the JSX, for it to be re-evaluted when the code changes.
\ No newline at end of file
+  return (
+    <button type="button" onClick={increment}>
+      {count()} 
+      {/* JSX is a tracking scope because it uses effects behind the scenes. Because of this, the code will register `count()` as a subscription when it is called. */}
```

Comment:
maybe saying that it "wraps expressions in effects" rather than a generic "uses effects" could be understandable, but I guess it should be fine either way.